src/CMakeLists.txt

```diff
@@ -244,6 +244,9 @@ add_library(stir_registries OBJECT ${STIR_REGISTRIES})
 # TODO, really should use stir_libs.cmake
 target_include_directories(stir_registries PRIVATE ${STIR_INCLUDE_DIR})
 target_include_directories(stir_registries PRIVATE ${Boost_INCLUDE_DIR})
+if (STIR_WITH_CUDA)
+  target_include_directories(stir_registries PRIVATE ${CUDA_INCLUDE_DIRS})
```

Comment:
I really hope we don't need `CUDA_INCLUDE_DIRS` for the registries. STIR .h files ideally have no CUDA includes at all. It's going to create trouble.

Reply:
I thought we'd want the target to be the registeries as we need the cuda_runtime.h when building cuda files that will be in the registry... Also I couldn't quite figure out where else to put the target_include_directories, where would you suggest?

Reply:
As far as I can see,`cuda_runtime.h should NOT be included in src/include/stir/recon_buildblock/CUDA/CudaRelativeDifferencePrior.h, but in the .cxx/.cu. That means that the registries wouldn't need to know about the CUDA include dir, therefore you don't need the `target_include_directories` at all. (The .cu files need it, but they get it by depending on CUDA targets).